### ISSUE TITLE
Reimagining the Best-practice Estimate method

### ISSUE THREAD

**Issue by PetrCala** [PetrCala/Diploma-Thesis#36]
## Overview

The best-practice method aims to evaluate the suggested estimate for each study (or author's setup) based on a defined best-practice. As of the current configuration, this is not being achieved, and should be modified sooner or later.

### Current configuration

Defining the *best-practice value* for each variable has an effect on the subjective (author's) best-practice estimate only. As for the dataset studies, a representative value is chosen for each variable, and this representative value is then multiplied (product-wise) with the BMA coefficients, yielding the estimate.

### Desired configuration

The value chosen for each study-variable pair should arise from a junction of two factors:
- Best-practice for that variable
- Actual values in the study

In other words, the code should look at the (for a given variable) available values in the study, choose one that best suits the best-practice restriction, and use that as the representative value.
